contextmanager.go

```diff
@@ -59,6 +59,22 @@ func newContextManager(label string) *contextManager {
 	return c
 }
 
+// Pick the first matching transfer syntax UID appeares in allTransferSyntaxes
+// - If one of the client proposed transfer syntaxes matches the server's preferred transfer syntax, then that transfer syntax is accepted.
+// - If the client does not propose a transfer syntax that matches the server's preferred transfer syntax, the first transfer syntax in the client's list of proposed syntaxes is accepted.
+// - If none of the proposed transfer syntaxes are supported, the server MUST send a bind_ack with all transfer syntaxes rejected.
```

Comment:
have we already handled this scenario in this MR?

Reply:
Not yet. I'm wondering if we should pick the first proposed transfer syntax or the first prefered transfer syntax instead.

Reply:
Please update the comment to reflect the current implementation then.

Reply:
done